foundation/foundation-configuration-store/src/test/java/com/boozallen/aissemble/configuration/store/LoadConfigurationsSteps.java

```diff
@@ -34,12 +43,19 @@ public class LoadConfigurationsSteps {
     private String baseURI;
     private String environmentURI;
     private Exception foundError;
-    private ValidatableResponse response;
+    private ValidatableResponse response; //change
 
+    private TestPropertyDao testPropertyDao;
+    private Set<Property> properties;
+    @InjectMocks
+    private ConfigStoreInitMock configStoreInitMock;
 
     @Before("@config-loader")
     public void setup() {
+        testPropertyDao = new TestPropertyDao();
         configLoader = new ConfigLoader();
+        configLoader.setPropertyDao(testPropertyDao);
+        configStoreInitMock = new ConfigStoreInitMock();
```

Comment:
A: we might be able to clean all these up since web server already loading all the properties

Reply:
removed those!